### PR TITLE
Add same offline charge animation as FP2.

### PR DESCRIPTION
**Pull request by rubencarneiro**
Now offline charging as the same animations as Fairphone 2

### CODE REVIEW

File: device.mk [hunk 1]
```diff
@@ -26,7 +26,7 @@ PRODUCT_COPY_FILES += \
     device/lge/hammerhead/init.hammerhead.rc:root/init.hammerhead.rc \
     device/lge/hammerhead/init.hammerhead.usb.rc:root/init.hammerhead.usb.rc \
     device/lge/hammerhead/fstab.hammerhead:root/fstab.hammerhead \
-    device/lge/hammerhead/ueventd.hammerhead.rc:root/ueventd.hammerhead.rc
+    device/lge/hammerhead/ueventd.hammerhead.rc:root/ueventd.hammerhead.rc \
```
Comment (Flohack74) [inline device.mk:29]: Why there is a \ at the end but no other line following? I think thats a mistake.

File: device.mk [hunk 2]
```diff
@@ -391,7 +390,18 @@ PRODUCT_COPY_FILES += \
...
-    device/lge/hammerhead/ubuntu/ofono.override:system/ubuntu/etc/init/ofono.override
+    device/lge/hammerhead/ubuntu/ofono.override:system/ubuntu/etc/init/ofono.override \
+    device/lge/hammerhead/ubuntu_overlay/system/etc/init/bluetooth-touch-hammerhead.conf:system/ubuntu-overlay/system/etc/init/bluetooth-touch-hammerhead.conf \
+    device/lge/hammerhead/ubuntu_overlay/system/etc/init/ofono.override:system/ubuntu-overlay/system/etc/init/ofono.override \
+    device/lge/hammerhead/ubuntu_overlay/system/etc/wifi/bcmdhd.cal:system/ubuntu-overlay/system/etc/wifi/bcmdhd.cal \
+    #device/lge/hammerhead/ubuntu_overlay/system/etc/acdbdata:system/ubuntu-overlay/system/etc/acdbdata \
+    #device/lge/hammerhead/ubuntu_overlay/system/etc/mixer_paths.xml:system/ubuntu-overlay/system/etc/mixer_paths.xml \
+    device/lge/hammerhead/ubuntu_overlay/system/usr/bin/hciattach:system/ubuntu-overlay/system/usr/bin/hciattach \
+    device/lge/hammerhead/ubuntu_overlay/system/usr/share/bluetooth-touch/hammerhead:system/ubuntu-overlay/system/usr/share/bluetooth-touch/hammerhead
+
```
Comment (Flohack74) [inline device.mk:401]: Your PR is named "charging animation" but here you also change a lot of the overlay stuff. This should not be part of the PR I think. Also, I think those files are already copied elsewhere, otherwise certain things would not work?

**Review by Flohack74 — CHANGES_REQUESTED** [review]
I left 2 comments. In general please keep PRs tidy and focused. Better make 2 PRs than one which contains multiple issues.

**Comment by UniversalSuperBox** [comment]
This was snuck in with https://github.com/ubports/android_device_lge_hammerhead/pull/18
